interface/src/app/analyze/analysis/sampleBin.js

```diff
@@ -133,7 +168,7 @@ function SampleBinCtrl($scope, $log, $uibModal, Sample, SampleBin) {
   $scope.sb = SampleBin;
 
   $scope.show = function() {
-    var modalInstance = $uibModal.open({
```

Comment:
Right. ESLint picked up on the fact that I was declaring the `var` and never using it. True to my philosophy of avoiding modals, this one will go away soon as well. Just trying to keep the pull requests focused by doing one thing at a time.